cmd/kubeadm/app/master/templates.go

```diff
@@ -78,10 +75,10 @@ spec:
           name: clusterinfo
           readOnly: true
       hostNetwork: true
-      # tolerations:
-      # - key: dedicated
-      #   value: master
-      #   effect: NoSchedule
+      tolerations:
+      - key: "dedicated"
```

Comment:
For kubeadm at least, look at the issues and https://github.com/kubernetes/kubernetes/pull/41835.
The taint key and value should equal the master label's key and value which for now seems to be `node-role.kubernetes.io/master` with no value `""`